manifests/params.pp

```diff
@@ -67,7 +67,7 @@
   $email_from = "no-reply@${::domain}"
   $email_enabled = false
 
-  $lazy_enabled = false
+  $manage_squid = true
```

Comment:
Its not that its 'on' or 'off'.  Its more that if you create a repo with a download policy of 'on_demand' or 'background' it requires squid to be configured and running.  

I believe pulp is still using the 'immediate' policy by default which means normal pulp users not using the new feature won't need squid